frontend/benefit/applicant/src/components/applications/forms/application/deMinimisAid/DeMinimisAidForm.tsx

```diff
@@ -61,6 +61,7 @@ const DeMinimisAidForm: React.FC = () => {
               name={fields.deMinimisAidIssueDate.name}
               label={fields.deMinimisAidIssueDate.label}
               placeholder={fields.deMinimisAidIssueDate.placeholder}
+              language="fi"
```

Comment:
I prefer if we define an `enum` of the supported languages and use its values instead (e.g: `Lang.FI`).

Reply:
yea we can. but it should we always the same value, independent of the locale we choose i guess

Reply:
yes